dist/api_data.py

```diff
@@ -1208,6 +1209,37 @@ def __ge__(self, other):
         characters are hexadecimal encoded.  These formats are compatible
         with the @ref util_dump and @ref util_load commands''',
         choices=['hex', 'json', 'print']),
+    Config('incremental', '', r'''
+        configure the cursor for block incremental backup usage. These formats
+        are only compatible with the backup data source; see @ref backup''',
+        type='category', subconfig=[
+        Config('enabled', 'false', r'''
+            whether to configure this backup as an incremental starting point''',
+            type='boolean'),
+        Config('file', '', r'''
+            the file name of interest when opening a duplicate incremental backup
+            cursor. That duplicate cursor will walk the block modifications relevant
+            to the given file name'''),
+        Config('force_stop', 'false', r'''
+            causes all block incremental backup information to be released. This is
+            on an open_cursor call and the resources will be released when this
+            cursor is closed. No other operations should be done on this open cursor''',
+            type='boolean'),
+        Config('granularity', '16MB', r'''
+            this setting manages the granularity of how WiredTiger maintains modification
+            maps internally. The larger the granularity, the smaller amount of information
+            we need to maintain''',
+            min='1MB', max='2GB'),
+        Config('src_id', '', r'''
```

Comment:
"src_id" -> "source_id"
The contraction isn't buying us a lot?

Reply:
I'm going to push back. I don't think the expanded word buys us a lot either. Both seem clear to me. That was the API published in the tech design document that everyone signed off on and no one suggested this then nor felt it was unclear. So it seems a little gratuitous. But I don't feel super strongly either. @milkie do you have an opinion about which is clearer or easier for MongoDB to use?

Reply:
fbm